testing/fake-dom.js

```diff
@@ -142,6 +142,10 @@ export class FakeWindow {
         }
       },
     });
+    // Polyfill performance.now() method as it would normally not be available
+    // in a non document environment. Used for tests using fakeWin that rely on
+    // the performance-impl service.
```

Comment:
This comment is slightly out of context and can be updated.

Reply:
what would you suggest?

Reply:
How about just `// Polyfill performance.now with Date.now.`. I think most of the current comment is obvious since it's living in `FakeWindow`.